hazelcast/src/main/java/com/hazelcast/nio/serialization/FieldType.java

```diff
@@ -55,4 +71,32 @@ public static FieldType get(byte type) {
         return ALL[type];
     }
 
+    public boolean isArrayType() {
+        return type >= PORTABLE_ARRAY.type;
```

Comment:
Thx Vassilis.I appreciate your input here - I guess it would have been a bit cleaner, but I don't really see a big improvement here. If you & others agree I would prefer to skip that fix.